doc/rst/usingchapel/prereqs.rst

```diff
@@ -65,8 +65,8 @@ We have used the following commands to install the above prerequisites:
 
   * Debian, Ubuntu::
 
-      sudo apt-get install gcc g++ m4 perl python3 python3-pip python3-venv python3-dev bash make mawk git pkg-config cmake llvm-12-dev llvm-12 llvm-12-tools clang-12 libclang-12-dev libclang-cpp12-dev libedit-dev
+      sudo apt-get install gcc g++ m4 perl python3 python3-pip python3-venv python3-dev bash make mawk git pkg-config cmake llvm-$LLVM_VERSION-dev llvm-$LLVM_VERSION llvm-$LLVM_VERSION-tools clang-$LLVM_VERSION libclang-$LLVM_VERSION-dev libclang-cpp$LLVM_VERSION-dev libedit-dev
```

Comment:
If it's not defined in the command, consider adding an example for setting `LLVM_VERSION` to something reasonable so that this remains a copy/paste operation.

Reply:
I was hoping to get out of needing to track versions of software on different linux distributions. Right now our preferred version would be `14`, but that isn't available at least on my Ubuntu system. So I could set it to `13`, but then we need to change it again once `14` is more available through `apt`.